hedera-mirror-rosetta/app/persistence/block/block_test.go

```diff
@@ -399,6 +430,30 @@ func TestShouldFailFindRecordFileByHashBlockNotFound(t *testing.T) {
 	assert.Nil(t, result)
 }
```

Comment:
I think that the upper test now should not be: `TestShouldFailFindRecordFileByHashBlockNotFound` but `TestShouldFailFindRecordFileByHashNegativeBlockIndex` or smth like that